luatest/justrun.lua

```diff
@@ -0,0 +1,172 @@
+--- Simple Tarantool runner and output catcher.
+--
+-- Sometimes it is necessary to run tarantool with particular arguments and
+-- verify its output. `luatest.server` provides a supervisor like
+-- interface: an instance is started, calls box.cfg() and we can
+-- communicate with it using net.box. Another helper,
+-- `test.interactive_tarantool`, aims to solve all the problems around
```

Comment:
I think we should give a clue what is this module. So I suggest this:

```
-- communicate with it using net.box. Another helper in tarantool/tarantool,
-- `test.interactive_tarantool`, aims ...
```

Reply:
Fixed.

```
diff --git a/luatest/justrun.lua b/luatest/justrun.lua
index 391fabf..d21ef8d 100644
--- a/luatest/justrun.lua
+++ b/luatest/justrun.lua
@@ -3,7 +3,7 @@
 -- Sometimes it is necessary to run tarantool with particular arguments and
 -- verify its output. `luatest.server` provides a supervisor like
 -- interface: an instance is started, calls box.cfg() and we can
--- communicate with it using net.box. Another helper,
+-- communicate with it using net.box. Another helper in tarantool/tarantool,
 -- `test.interactive_tarantool`, aims to solve all the problems around
 -- readline console and also provides ability to communicate with the
 -- instance interactively.
 ```